### ISSUE TITLE
GTK windows redraw and resize when they lose focus

### ISSUE THREAD

**Issue by onespaceman** [numixproject/numix-gtk-theme#362]
GTK: 3.20.3-1
DE: Budgie Desktop/Mutter

When a GTK window loses focus (ie nautilus, gnome-tweak-tool) it very quickly disappears/reappears and resizes to be bigger if the window is a small size. When it gains focus, it blinks again and resizes to its original size.

I'm able to get rid of the problem by commenting out in /gtk-3.20/scss/widgets/_window.scss lines 9-11 (the &:backdrop part), but I'm unsure of how to correctly modify it to fix the problem.

**Comment by khurshid-alam** [comment]
@onespaceman Is this the same as this? https://github.com/numixproject/numix-gtk-theme/commit/3cc526c0d6b1a59049b3d91ba490fb66b5281f1c#commitcomment-17108300

**Comment by onespaceman** [comment]
Yes, exactly like that.

**Comment by toupeira** [comment]
If you change the offsets to be the same it seems to fix the glitch:

``` diff
diff --git i/gtk-3.20/scss/widgets/_window.scss w/gtk-3.20/scss/widgets/_window.scss
index 7ca6ce0..3cea174 100755
--- i/gtk-3.20/scss/widgets/_window.scss
+++ w/gtk-3.20/scss/widgets/_window.scss
@@ -7,7 +7,7 @@
         box-shadow: 0 19px 38px rgba(0, 0, 0, .2), 0 15px 12px rgba(0, 0, 0, .1), 0 0 0 1px $wm_border_focused;

         &:backdrop {
-            box-shadow: 0 10px 20px rgba(0, 0, 0, .19), 0 6px 6px rgba(0, 0, 0, .23), 0 0 0 1px $wm_border_unfocused;
+            box-shadow: 0 19px 38px rgba(0, 0, 0, .19), 0 15px 12px rgba(0, 0, 0, .23), 0 0 0 1px $wm_border_unfocused;
         }
     }
```

**Comment by khurshid-alam** [comment]
This was fixed in 6532bce
